manifests/CPUID/HWMonitor/1.41.yaml

```diff
@@ -0,0 +1,14 @@
+Id: CPUID.HWMonitor
+Publisher: CPUID
+Name: HWMonitor
+Version: 1.41
+License: Copyright (C) CPUID
+InstallerType: EXE
+AppMoniker: hwmonitor
+Tags: CPU, GPU, HWMonitor, Monitor, Hardware
+Description: HWMonitor is a hardware monitoring program that reads PC systems main health sensors.
+Homepage: https://www.cpuid.com/softwares/hwmonitor.html
+Installers:
+  - Arch: x64
+    Url: http://download.cpuid.com/hwmonitor/hwmonitor_1.41.exe
+    Sha256: 2203e10b9277c0d74c3c12dc6808ea82f581fe01bd78a51b064d413eea9fd149
```

Comment:
```suggestion
    Sha256: 2203e10b9277c0d74c3c12dc6808ea82f581fe01bd78a51b064d413eea9fd149
    Switches:
      Silent: /VERYSILENT
      SilentWithProgress: /VERYSILENT
```

Reply:
Adding theses switches should fix the issue

Reply:
> Adding theses switches should fix the issue

Thank you very much!